defaults/main.yml

```diff
@@ -70,6 +70,13 @@ mysql_log: ""
 mysql_log_error: /var/log/mysql.err
 mysql_syslog_tag: mysql
 
+mysql_config_include_files: []
+# Full example:
```

Comment:
Just to follow the example set in many of my other roles, you can delete this line and the following, and just have the following two lines (with the actual example) commented, below the empty (`[]`) var.